service/gcs/bridge/connection.go

```diff
@@ -3,12 +3,13 @@ package bridge
 import (
 	"encoding/binary"
 
+	"github.com/Sirupsen/logrus"
 	"github.com/pkg/errors"
 
+	"io"
```

Comment:
You'll want to manually reorder this to the top. Ben just noticed the same thing in my change :)